isle_islandora_installer.sh

```diff
@@ -110,7 +110,8 @@ drush -y -u 1 en zip_importer
 drush -y -u 1 en islandora_basic_image
 drush -y -u 1 en islandora_bibliography
 drush -y -u 1 en islandora_compound_object
-drush -y -u 1 en islandora_google_scholar
+# Islandora_Google_Scholar has been refactored to Islandora_Scholar so the next line is no longer necessary.
+# drush -y -u 1 en islandora_google_scholar
```

Comment:
These lines can just come out as well - for the same reason - if we need to go back, we can look in Git. But we shouldn't need to go back with these.